homeassistant/components/frontend/www_static/polymer/dialogs/more-info-dialog.html

```diff
@@ -120,11 +136,11 @@
       }
     },
 
-    onIronOverlayOpened: function() {
+    onIronOverlayOpened: function() {      
```

Comment:
fixed, got a new laptop and hadn't enabled the trim whitespace on save option to sublime yet